.env.tpl

```diff
@@ -33,3 +33,11 @@ CYPRESS_ADMIN_USER_EMAIL=op://mdct_devs/mfp_secrets/CYPRESS_ADMIN_USER_EMAIL
 CYPRESS_ADMIN_USER_PASSWORD=op://mdct_devs/mfp_secrets/CYPRESS_ADMIN_USER_PASSWORD # pragma: allowlist secret
 CYPRESS_STATE_USER_EMAIL=op://mdct_devs/mfp_secrets/CYPRESS_STATE_USER_EMAIL
 CYPRESS_STATE_USER_PASSWORD=op://mdct_devs/mfp_secrets/CYPRESS_STATE_USER_PASSWORD # pragma: allowlist secret
+
+# db:seed
+SEED_ADMIN_PASSWORD=
+SEED_ADMIN_USER=
+SEED_STATE_PASSWORD=
+SEED_STATE_USER=
+SEED_STATE=
+SEED_STATE_NAME=
```

Comment:
suggestion: you could add these to the 1Password env for MFP, then it would populate these for you when you run the [new env commands](https://github.com/Enterprise-CMCS/macpro-mdct-mfp/pull/615)!

Reply:
I'll add these to 1password. Re: the env not being read. I just pulled the app anew with the install script and I ran into the `TypeError: Cannot read properties of undefined (reading 'split')` error also, but it was resolved when I `cd` out of the app and back into it. I'll see what can be done about not needing to do that step.

Reply:
hmm that is odd 🤔 I am getting the same error even when I try changing out and back into it